components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -400,6 +400,12 @@
     //Property for enabling scope sharing between APIs
     public static final String ENABLE_API_SCOPES_SHARING = "enable-api-scopes-sharing";
 
+    // Constants for obtaining organization claims
+    public static final String CHECK_ORGANIZATION_FROM_SAML_ASSERTION = "checkOrganizationFromSamlAssertion";
+    public static final String SAML2_SSO_AUTHENTICATOR_NAME = "SAML2SSOAuthenticator";
```

Comment:
Shouldn't this be "SAMLSSOAuthenticator"? IINM that's what we have in application-authentication.xml.

Reply:
No. If it is SAMLSSOAuthenticator, it refers to the application-authenticators.xml file in <AM_HOME>/repository/identity directory.
The correct file to be refered is authenticators.xml file in  <AM_HOME>/repository/conf/security directory. 
Where we can find following configuration.

<Authenticator name="SAML2SSOAuthenticator" disabled="false">
        <Priority>0</Priority>
        <Config>
            <Parameter name="LoginPage">/carbon/admin/login.jsp</Parameter>
            <Parameter name="ServiceProviderID">carbonServer</Parameter>
            <Parameter name="IdentityProviderSSOServiceURL">https://localhost:9444/samlsso</Parameter>
            <Parameter name="NameIDPolicyFormat">urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified</Parameter>
            <Parameter name="AssertionConsumerServiceURL">https://localhost:9443/acs</Parameter>
            <Parameter name="AssertionSignatureValidationEnabled">true</Parameter>
	    <Parameter name="OrganizationClaimAttribute">http://wso2.org/claims/organization</Parameter>
        </Config>